app/events/talk/page.tsx

```diff
@@ -0,0 +1,84 @@
+import comTalkImg from '@/assets/events/com-talk-img.png';
+import aprilComTalkImg from '@/assets/events/aprilComTalk.jpg';
+import watchIcon from '@/assets/icons/Watch 2.png';
+import calendarIcon from '@/assets/icons/Calender 2.png';
+
+import EventCard, { EventFeatCard } from '@/components/Events/EventCard';
+import Image from 'next/image';
+import { ContactCard } from '@/components/AboutUs/Cards';
+import { eventsEditionsData, eventsFeatsData } from '@/lib/data';
+import FAQ from '@/components/Events/FAQ';
+import EditionCard from '@/components/Events/EditionCard';
+
+const page = () => {
```

Comment:
_"page" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter._
```suggestion
const Page = () => {
```